test/Main.purs

```diff
@@ -4,19 +4,27 @@ import Prelude
 import Control.Apply ((*>))
 import Control.Alt ((<|>))
 import Control.Monad.Aff.AVar (makeVar', takeVar, putVar, AVar(), AVAR())
-import Control.Monad.Aff.Par (Par(..), runPar)
-import Control.Monad.Aff (attempt, later, later', launchAff, Aff())
-import Control.Monad.Eff.Class (liftEff)
-import Control.Monad.Error.Class (catchError)
+import Control.Monad.Aff (attempt, later, later', Aff)
+import Control.Monad.Eff (Eff)
+import Control.Monad.Eff.Ref (REF)
+import Control.Monad.Eff.Console (CONSOLE)
+import Control.Parallel.Class (parallel, runParallel)
 import Control.Plus (empty)
-import Data.Either (Either(..), either)
+import Data.Either as Either
 import Data.Functor (($>))
 import Data.Maybe (maybe, fromMaybe)
-import Data.Array (replicate, head, tail)
-import Test.Unit (runTest, test, assertFn)
+import Data.Array (head, tail)
+import Data.Unfoldable (replicate)
+import Test.Unit (test, suite)
+import Test.Unit.Assert (assert)
+import Test.Unit.Console (TESTOUTPUT)
+import Test.Unit.Main (runTest)
 
 import Control.Monad.Aff.Reattempt (reattempt)
 
+ignore :: forall m a. Monad m => m a -> m Unit
+ignore m = m >>= (\_ -> pure unit)
```

Comment:
![](https://cdn.meme.am/instances/500x/10484893.jpg)

Good catch; that's some trash I was experimenting with and forgot to delete.